WORKSPACE

```diff
@@ -144,3 +144,30 @@ switched_rules_by_language(
     cc = True,
     grpc = True,
 )
+
+http_archive(
+    name = "io_bazel_rules_go",
+    sha256 = "9fb16af4d4836c8222142e54c9efa0bb5fc562ffc893ce2abeac3e25daead144",
+    urls = [
+        "https://storage.googleapis.com/bazel-mirror/github.com/bazelbuild/rules_go/releases/download/0.19.0/rules_go-0.19.0.tar.gz",
+        "https://github.com/bazelbuild/rules_go/releases/download/0.19.0/rules_go-0.19.0.tar.gz",
+    ],
+)
+
+load("@io_bazel_rules_go//go:deps.bzl", "go_register_toolchains", "go_rules_dependencies")
+
+go_rules_dependencies()
+
+go_register_toolchains()
+
+http_archive(
+    name = "grpc_java",
+    strip_prefix = "grpc-java-master",
+    urls = ["https://github.com/grpc/grpc-java/archive/master.zip"],
+)
+
+http_archive(
+    name = "io_opencensus_proto",
+    strip_prefix = "opencensus-proto-master/src",
+    urls = ["https://github.com/panzhongxian/opencensus-proto/archive/master.zip"],
```

Comment:
I did the same thing to start with. We'll have to upstream these changes before submitting this PR.

Reply:
Do you mean that I should new a PR to opencensus-proto, or request another existing repos?

Reply:
> I did the same thing to start with. We'll have to upstream these changes before submitting this PR.

Do you mean that I should new a PR to opencensus-proto, or request another existing repos?

Reply:
I changed it to:
```python
urls = ["https://github.com/census-instrumentation/opencensus-proto/archive/master.zip"]
```

Reply:
You can keep it pointing at your repo for now.

We will have to change it back to the official repo before we merge this PR.

I have opened https://github.com/census-instrumentation/opencensus-proto/pull/212 to add the C++ proto targets upstream.

Reply:
Please change the URL to point to `github.com/census-instrumentation/opencensus-proto/archive` like you said, and use the C++ targets added in census-instrumentation/opencensus-proto#212